topics/imaging/tutorials/voronoi-segmentation/tutorial.md

```diff
@@ -0,0 +1,312 @@
+>    - *"Extract range"*: `All images`
+>    - *"Extract crop"*: `Full image`
+>    - *"Tile image"*: `No tiling`
+>    - *"Pyramid image"*: `No Pyramid`
+> 
+>    <!-- - **Important:** If using the tree image, choose the type of data as `zip`. --> 
+> 
+> 2. Rename the output to `single channel image`
+> 
+> 
+> 3. {% tool [Filter 2-D image with Scipy](toolshed.g2.bx.psu.edu/repos/imgteam/2d_simple_filter/ip_filter_standard/1.12.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input Image"*: `single channel image`
+>    - *"Filter type"*: `Gaussian`
+>        - *"Sigma"*: `3`
+> 
+> 4. Rename the output to `smoothed image`.
+> 
+>
+> 5. {% tool [Threshold image](toolshed.g2.bx.psu.edu/repos/imgteam/2d_auto_threshold/ip_threshold/0.18.1+galaxy3) %} with the following parameters:
+>    - {% icon param-file %} *"Input Image"*: `smoothed image`
+>    - *"Thresholding method"*: `Manual`
+>        - *"Threshold value"*: `3.0`
+> 
+> 6. Rename the output to `mask`.
+> 
+{: .hands_on}
+
+> <comment-title> How many channels does my image have? </comment-title>
+> Note: If providing your own image, you can check how many channels your image has with the tool `Show image info with Bioformats`. 
+> The number of channels is listed as e.g. `SizeC = 3` for the HeLa cell image or `SizeC = 3 (effectively 1)` for the tree image.
+{: .comment}
+
+> <comment-title> The value of Sigma and the Threshold value </comment-title>
+> Note: Generating a robust mask is harder for images with more noise. 
+> Since the tree image has more noise than the cell image, you may have to adjust the Sigma to achieve better results. 
+> You may also have to adjust the Threshold value in the last step. 
+{: .comment}
+
+
+> <question-title></question-title>
+>
+> 1. What is the purpose of the smoothing step? 
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The purpose of smoothing is to reduce noise. 
+> > For seed generation, noise might lead to false seeds where there is no object. 
+> > Smoothing also promotes connectedness within an object, where noise might make an object appear as two separate objects. 
+> > For mask generation, the same principles apply.
+> >
+> {: .solution}
+>
+{: .question}
+
+
+## Perform Voronoi segmentation based on the seeds 
+> <hands-on-title> Task description </hands-on-title>
+> We need to assign a unique label to each object in the seed image:
+> 1. {% tool [Convert binary image to label map](toolshed.g2.bx.psu.edu/repos/imgteam/binary2labelimage/ip_binary_to_labelimage/0.5+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Binary Image"*: `seeds`
+>    - *"Mode"*: `Connected component analysis`
+> 
+> 2. Rename the output to `label map`.
+>
+> 1. {% tool [Compute Voronoi tessellation with scikit-image](toolshed.g2.bx.psu.edu/repos/imgteam/voronoi_tesselation/voronoi_tessellation/0.22.0+galaxy3) %}. We use the label map to perform Voronoi segmentation. 
```

Comment:
The name of the tool is just `Compute Voronoi tessellation`, the `with scikit-image` part is the description:
```suggestion
> 1. {% tool [Compute Voronoi tessellation](toolshed.g2.bx.psu.edu/repos/imgteam/voronoi_tesselation/voronoi_tessellation/0.22.0+galaxy3) %}. We use the label map to perform Voronoi segmentation. 
```